site/content/docs/development/multi-cluster.md

```diff
@@ -0,0 +1,35 @@
+# Multi-Cluster
+
+## Overview
+It is often desirable to separate development and production environments; this can be achieved with multiple clusters.
+Cartographer helps pave a path to production across clusters with [ClusterSupplyChains](reference/workload.md#clustersupplychain)
+and [ClusterDeliveries](reference/deliverable.md#clusterdelivery), leveraging [GitOps](https://www.gitops.tech/).
+
+A ClusterSupplyChain can be installed in the development cluster to manage the path from source code to Kubernetes
+configuration in a Git repository. Meanwhile, a ClusterDelivery can be installed in the production cluster, which
+will pick up that configuration from the Git repository to be deployed and tested.
+
+![Multi-Cluster](../img/multi-cluster.jpg)
+
+### Example PR based flow
```

Comment:
@rawlingsj any strong opinions on this workflow before we draw up some diagrams?